src/libAtomVM/opcodesswitch.h

```diff
@@ -2995,6 +3000,30 @@ static bool maybe_call_native(Context *ctx, AtomString module_name, AtomString f
                 break;
             }
 
+            case OP_RAISE: {
+                uint8_t *pos = code + i;
+                int next_off = 1;
+                term arg1;
+                DECODE_COMPACT_TERM(arg1, code, i, next_off, next_off);
```

Comment:
maybe we should use a better name for arg1, so we can keep track of its meaning for future reference.

Reply:
Changed to "stacktrace" (which is unused) and "exc_value", which is (re) thrown